### PR TITLE
Feature/lakectl internal paging

### PR DESCRIPTION
**Pull request by johnnyaug**

### CODE REVIEW

File: cmd/lakectl/cmd/repo.go [hunk 1]
```diff
@@ -160,7 +137,7 @@ func init() {
 	repoCmd.AddCommand(repoCreateBareCmd)
 	repoCmd.AddCommand(repoDeleteCmd)
 
-	repoListCmd.Flags().Int("amount", -1, "how many results to return, or '-1' for default (used for pagination)")
+	repoListCmd.Flags().Int("amount", 100, "number of results to return")
```
Comment (nopcoder) [inline cmd/lakectl/cmd/repo.go:140]: constant. relevant to all the places in this change.

File: pkg/actions/db_task_result_iterator.go [hunk 2]
```diff
@@ -61,7 +61,7 @@ func (it *DBTaskResultIterator) maybeFetch() {
 		Select("run_id", "hook_run_id", "hook_id", "action_name", "start_time", "end_time", "passed").
 		From("actions_run_hooks").
 		Where(sq.Eq{"repository_id": it.repositoryID, "run_id": it.runID}).
-		Where(sq.Gt{"action_name": it.offset}).
+		Where(sq.Gt{"hook_run_id": it.offset}).
```
Comment (nopcoder) [inline pkg/actions/db_task_result_iterator.go:64]: good catch - I need to a test for this one

File: pkg/auth/service.go [hunk 3]
```diff
@@ -561,7 +561,7 @@ func (s *DBAuthService) ListGroupUsers(ctx context.Context, groupDisplayName str
 			WHERE
 				auth_groups.display_name = $1
 				AND auth_users.display_name > $2
-			ORDER BY auth_groups.display_name
+			ORDER BY auth_users.display_name
```
Comment (nopcoder) [inline pkg/auth/service.go:564]: how this one worked so far

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: pkg/auth/service.go (hunk 3 above)
Comment (johnnyaug) [reply pkg/auth/service.go:564]: it didn't :)

File: pkg/auth/service.go [hunk 4]
```diff
@@ -582,9 +582,10 @@ func (s *DBAuthService) ListGroupUsers(ctx context.Context, groupDisplayName str
 			WHERE
 				auth_groups.display_name = $1
 				AND auth_users.display_name > $2
-			ORDER BY auth_users.display_name
-			LIMIT $3`,
-			groupDisplayName, params.After, params.Amount+1)
+				AND auth_users.display_name like $3
```
Comment (nopcoder) [inline pkg/auth/service.go]: why listing needs to filter by prefix? we use this one as `after` which means you should get something that doesn't match the value's prefix.

Comment (nopcoder) [reply pkg/auth/service.go]: All good - didn't saw it is prefix.

**Review by nopcoder — APPROVED** [review]

**Pull request merged**